### ISSUE TITLE
Issue with dates

### ISSUE THREAD

**Issue by Shadow00Caster** [rickbergfalk/sqlpad#176]
ENV INFO
======
SqlPad: 2.2.0-beta1
MS SQL Server 2016, Windows 2012 R2, Latest Node, Running as service using nssm

ISSUE
======
I am running a query against a log table where TimeStart is datetime type.  When running the query in SSMS or something like Navicat, I get accurate responses.  When running in SqlPad, I get wrong dates.  I have tried setting the undocumented config variable of "localize" to false  in config-items.toml, this didn't appear to do anything (everything is in the same localization anyway).

CONFIG
======
```
[[configItems]]
key = "localize"
value = false
```

QUERY
========
```
SELECT
    COUNT(TimeStart) AS TotalSearches, 
    CONVERT(date, TimeStart) AS TheDate
FROM dbo.RequestDiagnostic
GROUP BY CONVERT(date, TimeStart)
ORDER BY TheDate ASC
```

DATA EXAMPLE
==========
![image](https://cloud.githubusercontent.com/assets/13611109/25299618/c828f504-26b5-11e7-92b8-dbf8a9d86c1a.png)

TABLE DESIGN
==========
![image](https://cloud.githubusercontent.com/assets/13611109/25299624/ddf3ef24-26b5-11e7-98a7-025befe5ebc8.png)

NAVICAT RESULTS
============
![image](https://cloud.githubusercontent.com/assets/13611109/25299487/85037520-26b4-11e7-89bc-a770bcad5ecd.png)

SQLPAD RESULTS
===========
![image](https://cloud.githubusercontent.com/assets/13611109/25299496/93c779a8-26b4-11e7-8b1b-4fb8301afe40.png)

Any help would be greatly appreciated!

**Comment by rickbergfalk** [comment]
Thanks for the report and all the info @Shadow00Caster! I'll have to look into this. 

Something is strange with the mssql dates being returned. I'm curious if dates from a 'datetimeoffset' column display correctly. What I think is happening is non-timezone dates are stored in the database, but somewhere between SQL Server --> node's mssql driver --> browser time zone gets added, messing with the date displayed.

The localize setting was something in v1 that was kind of a hack to fix the display of some values, so I took it out for v2 if I recall...

I'll have to see what I can find out.

**Comment by Shadow00Caster** [comment]
Thanks for the response @rickbergfalk, I figured it was some kind of localization issue; Makes sense the setting did nothing if it's gone :).

**Comment by rickbergfalk** [comment]
Fix in #212 - will be available in next release (planning for later this weekend)

Fix was confirmed against MySQL and Postgres (testing both local time and timestamp with time zone times in Postgres), which were both having the same issue. 

I _think_this should also fix the issue with MS SQL Server as well (all db query results were affected). 

If this isn't fixed please let me know and I'll try to get a SQL Server instance to dig in more.
